src/NzbDrone.Core/Indexers/TorrentRssIndexer/TorrentRssIndexer.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using FluentValidation.Results;
+using NLog;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Configuration;
+using NzbDrone.Core.Indexers.Exceptions;
+using NzbDrone.Core.Parser;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.Indexers.TorrentRssIndexer
+{
+    public class TorrentRssIndexer : HttpIndexerBase<TorrentRssIndexerSettings>
+    {
+        public override DownloadProtocol Protocol { get { return DownloadProtocol.Torrent; } }
+        public override Boolean SupportsSearch { get { return false; } }
+        public override Int32 PageSize { get { return 0; } }
+
+        public TorrentRssIndexer(IHttpClient httpClient, IConfigService configService, IParsingService parsingService, Logger logger)
+            : base(httpClient, configService, parsingService, logger)
+        {
+
+        }
+
+        public override IIndexerRequestGenerator GetRequestGenerator()
+        {
+            return new TorrentRssIndexerRequestGenerator() { Settings = Settings };
+        }
+
+        public override IParseIndexerResponse GetParser()
+        {
+            var factory = new TorrentRssParserFactory();
```

Comment:
Add this as ```ITorrentRssParserFactory``` to the constructor of ```TorrentRssIndexer```. That way it gets properly instantiated by tinyioc.